objects.js

```diff
@@ -4827,6 +4827,10 @@ SpriteMorph.prototype.mouseDownLeft = function () {
     return this.receiveUserInteraction('pressed');
 };
 
+SpriteMorph.prototype.mouseScroll = function (y) {
+    return this.receiveUserInteraction('scrolled-' + (y > 0 ? 'up' : 'down'));
```

Comment:
Hmmm... "up" and "down" is more of a problem than I expected it to be, especially since you can change it in your settings. This is going to be hard to explain to users. I wish we could get the actual data of the scroll wheel or touch pad...

Reply:
Call it "topward" and "bottomward."  :-)

Apple is just wrong about this, and we should take a stand for truth, justice, and the German way.

Reply:
What about something like scroll and reverse scroll?